R/utilities.r

```diff
@@ -302,6 +302,12 @@ has_name <- function(x) {
   !is.na(nms) & nms != ""
 }
 
+# Use chartr() for safety since toupper() fails to convert i to I in Turkish locale
+lower_ascii <- "abcdefghijklmnopqrstuvwxyz"
+upper_ascii <- "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+tolower_ascii <- function(x) chartr(upper_ascii, lower_ascii, x)
```

Comment:
Can we make this `to_lower_ascii()` and `to_upper_ascii()`?

And I think we should wrap `to_lower()` and `to_upper()` to throw an error, to make sure that this error doesn't arise again in the future.